zio-http/src/main/scala/zio/http/model/HeaderValue.scala

```diff
@@ -0,0 +1,90 @@
+package zio.http.model
+
+import zio.Chunk
+
+import scala.util.Try
+
+object HeaderValue {
```

Comment:
I guess HeaderValue will contain the big list of all the headers. Correct? In that case the file could become huge. Is not better create a package headervalues or something similar with all the headers there

Reply:
you are right. I have migrated to a package.